Compiled from "Exception.scala"
public class scala.util.control.Exception$Finally implements scala.util.control.Exception$Described {
  public final scala.Function0<scala.runtime.BoxedUnit> scala$util$control$Exception$Finally$$body;

  private final java.lang.String name;

  private java.lang.String scala$util$control$Exception$Described$$_desc;

  public java.lang.String scala$util$control$Exception$Described$$_desc();
    Code:
       0: aload_0
       1: getfield      #16                 // Field scala$util$control$Exception$Described$$_desc:Ljava/lang/String;
       4: areturn

  public void scala$util$control$Exception$Described$$_desc_$eq(java.lang.String);
    Code:
       0: aload_0
       1: aload_1
       2: putfield      #16                 // Field scala$util$control$Exception$Described$$_desc:Ljava/lang/String;
       5: return

  public java.lang.String desc();
    Code:
       0: aload_0
       1: invokestatic  #28                 // Method scala/util/control/Exception$Described$class.desc:(Lscala/util/control/Exception$Described;)Ljava/lang/String;
       4: areturn

  public scala.util.control.Exception$Described withDesc(java.lang.String);
    Code:
       0: aload_0
       1: aload_1
       2: invokestatic  #33                 // Method scala/util/control/Exception$Described$class.withDesc:(Lscala/util/control/Exception$Described;Ljava/lang/String;)Lscala/util/control/Exception$Described;
       5: areturn

  public java.lang.String toString();
    Code:
       0: aload_0
       1: invokestatic  #37                 // Method scala/util/control/Exception$Described$class.toString:(Lscala/util/control/Exception$Described;)Ljava/lang/String;
       4: areturn

  public java.lang.String name();
    Code:
       0: aload_0
       1: getfield      #39                 // Field name:Ljava/lang/String;
       4: areturn

  public scala.util.control.Exception$Finally and(scala.Function0<scala.runtime.BoxedUnit>);
    Code:
       0: new           #2                  // class scala/util/control/Exception$Finally
       3: dup
       4: new           #43                 // class scala/util/control/Exception$Finally$$anonfun$and$1
       7: dup
       8: aload_0
       9: aload_1
      10: invokespecial #47                 // Method scala/util/control/Exception$Finally$$anonfun$and$1."<init>":(Lscala/util/control/Exception$Finally;Lscala/Function0;)V
      13: invokespecial #50                 // Method "<init>":(Lscala/Function0;)V
      16: areturn

  public void invoke();
    Code:
       0: aload_0
       1: getfield      #55                 // Field scala$util$control$Exception$Finally$$body:Lscala/Function0;
       4: invokeinterface #60,  1           // InterfaceMethod scala/Function0.apply$mcV$sp:()V
       9: return

  public scala.util.control.Exception$Finally(scala.Function0<scala.runtime.BoxedUnit>);
    Code:
       0: aload_0
       1: aload_1
       2: putfield      #55                 // Field scala$util$control$Exception$Finally$$body:Lscala/Function0;
       5: aload_0
       6: invokespecial #62                 // Method java/lang/Object."<init>":()V
       9: aload_0
      10: invokestatic  #66                 // Method scala/util/control/Exception$Described$class.$init$:(Lscala/util/control/Exception$Described;)V
      13: aload_0
      14: ldc           #68                 // String Finally
      16: putfield      #39                 // Field name:Ljava/lang/String;
      19: return
}
